client/src/components/App/Content/InvestigationForm/InvestigationForm.test.tsx

```diff
@@ -28,6 +36,8 @@ describe('investigationForm tests', () => {
             });
         })
 
+        const epidemiologyNumber = '111';
+
         const expectedFirstSwal = {
```

Comment:
maybe add berfore all for the initializations

Reply:
In order to do so you must declare them outside and init them on the before all, and I think that declaring these variables outside is unrelevent for the other upcoming tests